src/models/github.js

```diff
@@ -11,6 +11,22 @@ class GitHub extends Fetcher {
     return this.get('notifications')
   }
 
+  getPandaRocketTasks() {
+    const urlPath = 'search/issues?q=team=identity-team-panda-rocket'
```

Comment:
I like that. Once we have a way to store the redux state locally, we can throw they team name in redux's state.